### PR TITLE
Create 16_how-do-i-change-the-privacy-settings-on-writings

### PR DESCRIPTION
**Pull request by DarklingMuse**
New FAQ topic from review in FAQ document

### CODE REVIEW

File: 03_pictures-videos-and-writings/16_how-do-i-change-the-privacy-settings-on-writings [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# How do I change the privacy settings on one of my Writings?
+
+When you create a Writing on FetLife the default privacy setting is set to ‘FetLifers’ - meaning everyone on FetLife will be able to view it!
+
+You can change this before you post your Writing by selecting the ‘Friends-Only’ option from the drop down menu at the bottom of the new Writing box.
```
Comment (sylee-) [inline 03_pictures-videos-and-writings/16_how-do-i-change-the-privacy-settings-on-writings:5]: "drop-down" instead of "drop down"

**Review by sylee- — CHANGES_REQUESTED** [review]
I didn't mean to star a review.  I meant to add a comment and now I can't figure out how to get rid of the review.  :(

**Comment by DarklingMuse** [comment]
I've edited this to remove mentions of drop down menus / too many specifics so that it is still relevant if things move around / it's concise... I think it still makes sense? 😅

**Comment by DarklingMuse** [comment]
replaced by https://github.com/fetlife/help/pull/126

**Pull request closed without merging**